scripts/upgrade.sh

```diff
@@ -92,11 +92,8 @@ fi
 # commander - v13 has backwards-incompatible changes which require a decent amount of refactoring to get our current code to work. We are considering migrating off of commander but for now we should just freeze it
 # eslint - version 9+ conflicts with Next.js plugins, holding back until fixed
 # jose - version 6+ requires ESM (depending on the precise NodeJS version), holding back until server supports ESM
-# react-router-dom - version 7+ has breaking changes, will fix separately
 # node-fetch - version 3+ requires ESM, holding back until server supports ESM
-# rimraf - version 6+ requires Node 20+, holding back until Medplum v4
-# supertest - version 7+ incompatible with superwstest, waiting for fix
-MAJOR_EXCLUDE="@types/express @types/node @types/react @types/react-dom commander eslint jose react-router-dom node-fetch rimraf supertest"
+MAJOR_EXCLUDE="@types/express @types/node @types/react @types/react-dom commander eslint jose node-fetch"
```

Comment:
Can probably remove `@types/react-dom` too?

Reply:
Also did we investigate removing `node-fetch` now that `fetch` has stabilized for `Node20`? or I guess we may have to support it because of bots...

Reply:
> Can probably remove `@types/react-dom` too?

I figured we should treat that the same as `react`?  Might be worth a try.

> Also did we investigate removing node-fetch now that fetch has stabilized for Node20? 

I've been keeping my notes here:  https://github.com/medplum/medplum/issues/1563

Node 20 does seem like the last blocker.

I think that testing is quite a bit different with built-in fetch, and you need to use the undici `MockAgent` or something.  Maybe?  I assume that's a non-trivial change.